pkg/kp/service.go

```diff
@@ -38,3 +39,16 @@ func (c consulStore) RegisterService(manifest pods.Manifest, caPath string) erro
 
 	return c.client.Agent().ServiceRegister(podService)
 }
+
+// Go version of http status check
+func HttpStatusCheck(node string, port int) *http.Response {
+	url := fmt.Sprintf("http://%s:%d/_status", node, port)
+	resp, _ := http.Get(url)
+	return resp
+}
+
+func HttpsStatusCheck(node string, port int) *http.Response {
+	url := fmt.Sprintf("https://%s:%d/_status", node, port)
+	resp, _ := http.Get(url)
```

Comment:
if Get has a non-2xx response it returns an err. If figured we would rather handle such an error by looking at the response.